src/v2/ion/RemediationConstants.js

```diff
@@ -127,6 +127,23 @@ const HINTS = {
   CAPTCHA: 'captcha',
 };
 
+const TERMINAL_VIEWS = [
+  FORMS.TERMINAL,
+  FORMS.DEVICE_ENROLLMENT_TERMINAL,
+];
+
+const SUCCESS_VIEWS = [
+  FORMS.SUCCESS_REDIRECT,
+]
+
+const SHOULD_NOT_SAVE_STATE_HANDLE =
```

Comment:
This name may be a bit misleading, as "should" indicates it *may have*. What about `FORMS_WITHOUT_STATE_HANDLE`, to adhere to our convention of `FORM_(with|without)_${attribute}`?